its/tests/src/test/java/its/SonarQubeDeveloperEditionTests.java

```diff
@@ -179,7 +179,15 @@ void start(@TempDir Path sonarUserHome) {
         // Needed to have the global extension plugin loaded
         .addEnabledLanguage(Language.COBOL)
         .addEnabledLanguage(Language.GO)
+        .addEnabledLanguage(Language.CLOUDFORMATION)
+        .addEnabledLanguage(Language.DOCKER)
+        .addEnabledLanguage(Language.KUBERNETES)
+        .addEnabledLanguage(Language.TERRAFORM)
         .useEmbeddedPlugin(Language.GO.getPluginKey(), PluginLocator.getGoPluginPath())
+        .useEmbeddedPlugin(Language.CLOUDFORMATION.getPluginKey(), PluginLocator.getIacPluginPath())
+        .useEmbeddedPlugin(Language.DOCKER.getPluginKey(), PluginLocator.getIacPluginPath())
+        .useEmbeddedPlugin(Language.KUBERNETES.getPluginKey(), PluginLocator.getIacPluginPath())
+        .useEmbeddedPlugin(Language.TERRAFORM.getPluginKey(), PluginLocator.getIacPluginPath())
```

Comment:
You normally only need one of those, since the plugin key will be the same for each of those languages